addons/block_code/ui/picker/categories/category_factory.gd

```diff
@@ -192,3 +192,93 @@ static func add_to_categories(main: Array[BlockCategory], addition: Array[BlockC
 			main.append(add_category)
 
 	return main
+
+
+static func built_in_type_to_block_type(type: Variant.Type):
+	match type:
+		TYPE_BOOL:
+			return Types.BlockType.BOOL
+		TYPE_INT:
+			return Types.BlockType.INT
+		TYPE_FLOAT:
+			return Types.BlockType.FLOAT
+		TYPE_STRING:
+			return Types.BlockType.STRING
+		TYPE_VECTOR2:
+			return Types.BlockType.VECTOR2
```

Comment:
I think there shouldn't be block types that map to Variant types, we can use Variant types directly (see my spec). All these blocks can be a single Variant block that has Variant type as property. But good to keep this for now!

Reply:
I agree, the current typing system needs some work, but it is useful to have the BlockType stay as it is now, since there isn't really a type for `EXECUTE` in `Variant.Type`.